orchagent/portsorch.cpp

```diff
@@ -867,62 +868,200 @@ bool PortsOrch::setPortPfcAsym(Port &port, string pfc_asym)
     return true;
 }
 
-bool PortsOrch::createBindAclTableGroup(sai_object_id_t id, sai_object_id_t &group_oid, acl_stage_type_t acl_stage)
+/*
+ * Name: bindUnbindAclTableGroup
+ *
+ * Description:
+ *     To bind a port to ACL table we need to do two things.
+ *     1. Create ACL table member, which maps 
+ *        ACL table group OID --> ACL table OID
+ *     2. Set ACL table group OID as value port attribute.
+ *
+ *      This function performs the second step of binding.
+ *
+ *      Also, while unbinding we use this function to 
+ *      set port attribute value to SAI_NULL_OBJECT_ID
+ *
+ *      Port attribute name is derived from port type
+ *
+ * Return: true on success, false on failure
+ */
+bool PortsOrch::bindUnbindAclTableGroup(Port &port,
```

Comment:
I agree with your decision to combine the bind and unbind into one method, but I think maybe it would make sense to include "bindAclTable" and "unbindAclTable" wrappers just for clarity. I think it would be more readable than just having the boolean flag at the end of the method call.

Reply:
now that I've had a closer look I realize this is pretty much what you did below. :P is this used anywhere outside of portsorch?

Reply:
Here what I practice, if two functions have 80% of the lines common, merge them, because you have to fix bugs in only one place and its also easy to read. If they both start diverging in future split them. So, I prefer keep it as one function for now.

Reply:
I agree with you, I wasn't suggesting splitting them up, rather just having wrappers for bind/unbind. However, that is pretty much how you ended up using this function, so my original comment isn't relevant here.